src/cls/rbd/cls_rbd_types.cc

```diff
@@ -753,13 +754,15 @@ void MirrorNonPrimarySnapshotNamespace::decode(bufferlist::const_iterator& it) {
   decode(primary_snap_id, it);
   decode(copied, it);
   decode(last_copied_object_number, it);
+  decode(snap_seqs, it);
 }
 
 void MirrorNonPrimarySnapshotNamespace::dump(Formatter *f) const {
   f->dump_string("primary_mirror_uuid", primary_mirror_uuid);
   f->dump_unsigned("primary_snap_id", primary_snap_id);
   f->dump_bool("copied", copied);
   f->dump_unsigned("last_copied_object_number", last_copied_object_number);
+  f->dump_stream("snap_seqs") << snap_seqs;
```

Comment:
nit: don't we want to update `operator<<(std::ostream&, const MirrorNonPrimarySnapshotNamespace&)` too?

Reply:
Added